service/raft/raft_sys_table_storage.cc

```diff
@@ -89,6 +89,28 @@ future<raft::index_t> raft_sys_table_storage::load_commit_idx() {
     co_return raft::index_t(static_row.get_or<int64_t>("commit_idx", raft::index_t{}.value()));
 }
 
+namespace {
+    class managed_bytes_view_fragment_iterator {
```

Comment:
Is this a valid approach? I couldn't find an existing class that handles this.

Reply:
Or @avikivity, fragment_range_view

Reply:
thanks, updated